atm.js

```diff
@@ -1,3 +1,118 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+     console.log("Hola hola hola");
+
+// CHECKING DEPOSIT
+
+$("#checking > .deposit").on("click", function(evt){
+    // Prevent default action
+    evt.preventDefault();
+
+  // get user input && convert to float with 2 decimal placess
+    var newInput = $("#checking > .input").val()
+    var floatnewInput = parseFloat(newInput).toFixed(2);
+    console.log(floatnewInput);
+
+    // update total
+      //  get current total
+    var total = $("#checking > .balance").html()
+    var numerictotal = parseFloat(total.split("$")[1])
+    var actualtotal = parseFloat(floatnewInput) + numerictotal
+    $("#checking > .balance").html("$ " + actualtotal.toFixed(2));
+    console.log()
+    $("#checking > .input").val("")
+
+    // Turn checking box to red if balance reaches 0
+    if (actualtotal == 0) {
+      $("#checking").removeClass("account").addClass("zero")
+    }
+    })
+
+// CHECKING WITHDRAWAL
+
+$("#checking > .withdraw").on("click", function(evt){
+    // Prevent default action
+    evt.preventDefault();
+
+  // get user input && convert to float with 2 decimal placess
+    var newInput = $("#checking > .input").val()
+    var floatnewInput = parseFloat(newInput).toFixed(2);
+    console.log(floatnewInput);
+
+    // update total
+      //  get current total
+    var total = $("#checking > .balance").html()
+    var numerictotal = parseFloat(total.split("$")[1])
+    var actualtotal = numerictotal - parseFloat(floatnewInput)
+    $("#checking > .balance").html("$ " + actualtotal.toFixed(2));
```

Comment:
So this code allows you balance to go negative. I would have included a conditional for this to be executed only if the resulting total would be greater than or equal to 0.